### PR TITLE
Add Fixer descriptions

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -532,13 +533,13 @@ Choose from the list of available rules:
    | *Rule is: risky.*
 
 * **random_api_migration** [@PHP70Migration, @PHP71Migration]
-   | Replaces rand, srand, getrandmax functions calls with their mt_*
-   | analogs.
+   | Replaces `rand`, `mt_rand`, `srand`, `getrandmax` functions calls with
+   | their mt_* analogs.
```
Comment (localheinz) [inline README.rst]: Should `mt_*` also be embedded within back ticks?

File: src/Fixer/Alias/NoMixedEchoPrintFixer.php [hunk 2]
```diff
@@ -71,9 +73,18 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
...
     {
-        return $tokens->isTokenKindFound($this->candidateTokenType);
+        return new FixerDefinition(
+            'Either language construct `print` or `echo` should be used.',
+            array(
+                new CodeSample('<?php print \'example\';'),
+                new CodeSample('<?php echo(\'example\');', array('use' => 'print')),
+            ),
+            null,
+            'The fixer can be configured to change `print` to `echo` `[\'use\' => \'echo\']` or `echo` to `print` `[\'use\' => \'print\']`.',
```
Comment (localheinz) [inline src/Fixer/Alias/NoMixedEchoPrintFixer.php]: How about using double quotes here so we don't need to escape single quotes?

File: src/Fixer/ArrayNotation/WhitespaceAfterCommaInArrayFixer.php [hunk 3]
```diff
@@ -45,9 +39,20 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
...
+        return new FixerDefinition(
+            'In array declaration, there MUST be a whitespace after each comma.',
+            array(new CodeSample("<?php\n\$sample = array(1,'a',\$b,);"))
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
```
Comment (localheinz) [inline src/Fixer/ArrayNotation/WhitespaceAfterCommaInArrayFixer.php]: Wondering, should the reordering be done by a fixer? 😄

Comment (SpacePossum) [inline src/Fixer/ArrayNotation/WhitespaceAfterCommaInArrayFixer.php]: ordered_class_elements ? ;)

File: README.rst (hunk 1 above)
Comment (SpacePossum) [inline README.rst]: :+1:

File: src/Fixer/Alias/NoMixedEchoPrintFixer.php (hunk 2 above)
Comment (SpacePossum) [inline src/Fixer/Alias/NoMixedEchoPrintFixer.php]: :+1:

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Comment by keradus** [comment]
Great work here!

**Pull request closed without merging**